toxcore/Messenger.h

```diff
@@ -796,6 +796,9 @@ uint8_t *messenger_save(const Messenger *m, uint8_t *data);
 
 /* Load a state section.
  *
+ * @param data Data to load
```

Comment:
Add "." at the end (or remove it from all the others).

Reply:
done